packages/salesforcedx-vscode-core/src/services/sdr/componentSetUtils.ts

```diff
@@ -54,16 +54,22 @@ export async function setSourceApiVersion(componentSet: ComponentSet): Promise<v
 
   // Next, if it still is not set, set it to the highest API version supported by the target org.
   if (!sourceApiVersion) {
-    const orgApiVersion = await getOrgApiVersion();
+    const orgApiVersion = await componentSetUtils.getOrgApiVersion();
     sourceApiVersion = orgApiVersion;
   }
 
   componentSet.sourceApiVersion = sourceApiVersion;
 }
 
-export async function getOrgApiVersion(): Promise<string> {
+async function getOrgApiVersion(): Promise<string> {
   const connection = await WorkspaceContext.getInstance().getConnection();
   const apiVersion = connection.getApiVersion();
 
   return apiVersion;
 }
+
+export const componentSetUtils = {
```

Comment:
Changed the way the functions in componentSetUtils.ts are exported in order to write unit tests for them.